genesyscloud/architect_datatable_row/resource_genesyscloud_architect_datatable_row_proxy.go

```diff
@@ -99,6 +107,7 @@ func getAllArchitectDatatableFn(ctx context.Context, p *architectDatatableRowPro
 
 	for _, table := range *tables.Entities {
 		totalRecords = append(totalRecords, table)
+		rc.SetCache(p.dataTableCache, *table.Id, *ConvertDatatable(table))
```

Comment:
Just out of curiosity.  Why are you passing in the cache into this method?  I think on the original example we had the happening directly on the cache.

Reply:
Just added a wrapper method , 

it removes the dependency of tfexporter from all the proxies.
also removes few boiler plate code like exporter checks and added few log statements added when there is a cache miss.